doc/1/api/controllers/server/public-api/index.md

```diff
@@ -0,0 +1,82 @@
+---
+code: true
+type: page
+title: publicApi
+---
+
+# publicApi
+
+
+Returns available API, including Kuzzle API and plugins API.  
+
+::: warning
+This route is used by the HTTP protocols of the SDKs to build requests based on the controller and action names.  
```

Comment:
(debate/optional) About plugins, shouldn't we have an option for that route allowing to either disable plugins routes, or maybe to hide some routes, or both?

I feel that might be needed for multi-tenant applications that need to hide routes created dynamically for their clients.

Reply:
Also, shouldn't the SDKs re-invoke that route upon a successful login attempt? So they can work if this route is closed for anonymous users, and opened for logged in ones, making this warning valid only if rights are too restricted, but not necessarily for anon users?

Reply:
1) We discussed this point in the workshop and we agreed that if a plugin developer want to use additional routes in a SDK with the HTTP protocol, he should add theses routes himself. 
But As I said in the workshop, I agree with you on the need to hide some plugin routes.

2) Yes we can do that.

Reply:
Ah yes, I forgot about the privacy flag added to plugins http routes. I apologize.